definitions/mlops-endpoint/definition.yml

```diff
@@ -0,0 +1,22 @@
+domain: MLOPS
+type: ENDPOINT
```

Comment:
Are endpoints a Sagemaker-specific concept? I'm wondering if we should use SM_ENDPOINT or something like that.

We have another ENDPOINT type in the EXT- domain.  This will not cause problems anywhere, but for historical reasons telemetry doesn't get stamped with the domain, so this case will generate ambiguities when we process them.  It won't break anything, but I'm wondering if it's worth trying to avoid it anyway.

Reply:
changed it to MACHINE_LEARNING_ENDPOINT (we would want to include endpoints from other sources in the future)